pkg/cloud/vsphere/services/cloudprovider/addons.go

```diff
@@ -0,0 +1,277 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudprovider
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	cloudProviderImage = "gcr.io/cloud-provider-vsphere/cpi/release/manager:latest"
```

Comment:
For the sake of simplicity I hardcoded this to `latest` but I think we should figure out how to pick a vSphere CCM version w.r.t to CAPV.

Reply:
Actually, in a conversation I had with @akutz we agreed to make the version configurable via the cloud configuration field.